core/trino-main/src/main/java/io/trino/connector/CoordinatorDynamicCatalogManager.java

```diff
@@ -137,7 +137,7 @@ public void loadInitialCatalogs()
                                     CatalogConnector newCatalog = catalogFactory.createCatalog(catalog);
                                     activeCatalogs.put(catalog.getCatalogHandle().getCatalogName(), newCatalog.getCatalog());
                                     allCatalogs.put(catalog.getCatalogHandle(), newCatalog);
-                                    log.info("-- Added catalog %s using connector %s --", storedCatalog.getName(), catalog.getConnectorName());
+                                    log.debug("-- Added catalog %s using connector %s --", storedCatalog.getName(), catalog.getConnectorName());
```

Comment:
this is useful for admins and people who help users to verify if a particular catalog was indeed loaded or not.

The prune and `createCatalog` ones are fine to change to debug though.